opensrp-app/src/test/java/org/smartregister/util/JsonFormUtilsTest.java

```diff
@@ -1504,4 +1504,21 @@ public void addObservationAddsGpsObservationsToEvent() throws Exception {
             }
         }
     }
+
+    @Test
+    public void testCreateObservationForNativeRadioShouldAddKeyValuePairsToObservation() throws Exception {
+        String strJsonObj = "{\"key\":\"not_good\",\"openmrs_entity_parent\":\"\",\"openmrs_entity\":\"\",\"openmrs_entity_id\":\"\",\"type\":\"native_radio\",\"label\":\"Quel est le problème avec ce produit ?\",\"label_text_style\":\"bold\",\"options\":[{\"key\":\"worn_broken\",\"text\":\"Usé, endommagé, ou cassé\",\"openmrs_entity_parent\":\"\",\"openmrs_entity\":\"\",\"openmrs_entity_id\":\"\"},{\"key\":\"expired\",\"text\":\"Expiré\",\"openmrs_entity_parent\":\"\",\"openmrs_entity\":\"\",\"openmrs_entity_id\":\"\"},{\"key\":\"parts_missing\",\"text\":\"Pièces manquantes\",\"openmrs_entity_parent\":\"\",\"openmrs_entity\":\"\",\"openmrs_entity_id\":\"\"},{\"key\":\"other\",\"text\":\"Autre (préciser)\",\"openmrs_entity_parent\":\"\",\"openmrs_entity\":\"\",\"openmrs_entity_id\":\"\"}],\"v_required\":{\"value\":true,\"err\":\"Ce champ est requis\"},\"relevance\":{\"step1:flag_problem\":{\"ex-checkbox\":[{\"or\":[\"not_good\"]}]}},\"is-rule-check\":false,\"is_visible\":true,\"value\":\"worn_broken\"}";
+        JSONObject jsonObject = new JSONObject(strJsonObj);
+        Event event = new Event();
+        String value = "worn_broken";
+        Whitebox.invokeMethod(JsonFormUtils.class, "createObservation", event, jsonObject, value);
+        List<Obs> obsList = event.getObs();
+        assertEquals(1, obsList.size());
+        Obs obs = obsList.get(0);
+        assertNotNull(obs.getKeyValPairs());
+        assertEquals(1, obs.getKeyValPairs().size());
+        assertNotNull(obs.getKeyValPairs().get("worn_broken"));
```

Comment:
`obs.getKeyValPairs().get("worn_broken")` is repeated.  `worn_broken` is already defined https://github.com/OpenSRP/opensrp-client-core/pull/770/files#diff-1741b1114c81ba71c2f6c91f4bfc62668c65737537bf1f0ee15f27cd58192c9dR1513.